plugins/org.jboss.tools.openshift.ui/src/org/jboss/tools/openshift/internal/ui/wizard/connection/BasicAuthenticationDetailView.java

```diff
@@ -119,6 +123,12 @@ public void setSelectedConnection(IConnection conn) {
 	@Override
 	public void onInVisible(IObservableValue detailsViewModel, DataBindingContext dbc) {
 		disposeBindings();
+		setCheckboxVisible(false);
+	}
+
+	private void setCheckboxVisible(boolean b) {
+		chkRememberToken.setVisible(b);
+		((GridData)chkRememberToken.getLayoutData()).heightHint = b ? SWT.DEFAULT : 1;
```

Comment:
heightHint = -1 makes widget full heighgt. heightHint = 1 makes it minimum height. Without that there will be an empty space. You may try to run it without this layout change, if it works for you, then, yes, it may be GTK3.